airflow/providers/google/cloud/operators/dataproc_metastore.py

```diff
@@ -144,10 +144,10 @@ def get_link(self, operator, dttm):
         )
 
 
-class DataprocMetastoreStorageLink(BaseOperatorLink):
-    """Helper class for constructing GCS Storage link for Dataproc Metastore Export Metadata"""
+class StorageLink(BaseOperatorLink):
```

Comment:
I think we should move this to the `airflow/providers/google/common/links/storage.py` I think it can be used in future in different operators